jupyterlab_bigquery/setup.py

```diff
@@ -55,5 +55,8 @@
         "google-cloud-bigquery~=1.25.0",
```

Comment:
Can you convert all of these from `~=` to `>=`? we had an issue with dependencies downgrading.